lab-configs.yaml

```diff
@@ -24,6 +24,11 @@ labs:
             - boot-nfs
             - simple
             - sleep
+      - whitelist:
```

Comment:
Do you need to say "- whitelist:" again ?

Reply:
No it's not needed actually, I've just push a fix for that.  It works in any case, but this it's a bit simpler to have only one whitelist filter.